src/main/java/com/softserveinc/ita/pageobjects/ProductModel.java

```diff
@@ -1,11 +1,11 @@
-package com.softserveinc.ita.models.repos;
+package com.softserveinc.ita.pageobjects;
 
-import com.softserveinc.ita.models.ProductModel;
+import com.softserveinc.ita.models.Product;
 
 import static com.codeborne.selenide.Selenide.$x;
 import static com.softserveinc.ita.utils.NumberUtil.parseIntPrice;
 
-public class ProductRepo {
+public class ProductModel {
```

Comment:
why this class is needed? if it's related to a ProductPage ?

Reply:
where i have to move them?

Reply:
all these methods probably should be in ProductPage, as I understand all operations occur there